backend/app/views/static_pages/root.html.erb

```diff
@@ -1 +1,6 @@
-<div id="root"></div>
+<div id="root">
+<h1>React is not working</h1>
```

Comment:
Maybe a friendlier message here... if users are on a slow internet connection or slow cpu, they will actually see this message for an extended period of time while the JavaScript assets are downloaded and then executed. Common convention is to place “loading...” here

Reply:
That makes sense. Have changed it.